gym/exporter_main_v2.py

```diff
@@ -0,0 +1,159 @@
+# Lint as: python2, python3
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+r"""Tool to export an object detection model for inference.
+
+Prepares an object detection tensorflow graph for inference using model
+configuration and a trained checkpoint. Outputs associated checkpoint files,
+a SavedModel, and a copy of the model config.
+
+The inference graph contains one of three input nodes depending on the user
+specified option.
+  * `image_tensor`: Accepts a uint8 4-D tensor of shape [1, None, None, 3]
```

Comment:
Something odd on the documentation. You mention a "uint8" but the options you define on line 108 are strings.

Reply:
This module is part of tensorflow api for object  detection,  I didn't implement it myself  ... 

And about  the line 108 is defining command  lines arguments  

https://abseil.io/docs/python/guides/flags

Reply:
Ok, if it was not implemented by you then it's not our problem, I know about the flags, my question is about misalignment between the code (line 108) and the documentation at this line. I guess maybe they have changed the code at a certain point in time but forgot to change the documentation. 

Actually, now that I see again, this is even more different than I though. Ok, nevermind, let's move on on this.